interview-questions.md

```diff
@@ -4,47 +4,47 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: It is a method that takes in any number of state objects and allows them to be reffered to by shorthand for the duration of the current method you're in.
```

Comment:
Pretty close! I know you were able to determine a better answer in your researched answer.